resources/adobe-experience-platform-mini-programs-sdk/README.md

```diff
@@ -57,7 +57,7 @@ The SDK is available as read-only open source and is distributed with an Apache
 * 从服务器端获取或者小程序端自动生成
 
 {% hint style="info" %}
-有关Analytics Visitor ID的信息，请参阅 [Adobe Analytics implementation guide](https://docs.adobe.com/content/help/en/analytics/implementation/javascript-implementation/unique-visitors/visid-analytics.html#concept_74F6B4B9B2FA415AB5D029A1F8F099BC)
+有关Analytics Visitor ID的信息，请参阅 [Adobe Analytics implementation guide](https://experienceleague.adobe.com/docs/analytics/implementation/js/xdevice-visid/xdevice-connecting.html?lang=en)
```

Comment:
The page on this link says this is not the recommended method. Maybe we can clarify if this is the correct link?